### PR TITLE
Climate strike setting

### PR DESCRIPTION
**Pull request by scottsweb**
#### Changes proposed in this Pull Request

Screenshot_2019-09-10 Site Settings ‹ Jack Lenox — WordPress com

Introduce a new setting for controlling the climate strike banner (pb6Nl-d71-p2).

Copy review in progress.

#### Testing instructions

**Dependencies**

D32643-code
D32647-code

With both the API and plugin in place and your site and the API sandboxed, you can test this PR on any WordPress.com site. The setting will not show for Jetpack sites which can install the [plugin directly](https://href.li/?https://wordpress.org/plugins/digital-climate-strike-wp/).

### CODE REVIEW

**Comment by ockham** [comment]
Nice!

Noting that this is modeled after #20748. It will require some backend work akin to r166738-wpcom and r166747-wpcom, respectively :slightly_smiling_face:

File: client/my-sites/site-settings/form-general.jsx [hunk 1]
```diff
@@ -353,6 +354,74 @@ export class SiteSettingsFormGeneral extends Component {
...
+						<CompactFormToggle
+							checked={ !! fields.climatestrike }
+							disabled={ isRequestingSettings }
+							onChange={ handleToggle( 'climatestrike' ) }
+						>
+							{ translate(
+								'This September, millions of us will walk out of our workplaces and homes to join ' +
+									'young climate strikers on the streets and {{climateStrikeLink}}demand an end to the age ' +
+									'of fossil fuels{{/climateStrikeLink}}. Show your solidarity with a {{digitalClimateStrikeLink}}digital ' +
+									'climate strike banner{{/digitalClimateStrikeLink}}. On September 20th the banner will expand to a dissmissable overlay.',
```
Comment (Aurorum) [inline client/my-sites/site-settings/form-general.jsx]: Just wanted to note a typo that I spotted. :)

```suggestion
									'climate strike banner{{/digitalClimateStrikeLink}}. On September 20th the banner will expand to a dismissible overlay.',
```

Comment (scottsweb) [reply client/my-sites/site-settings/form-general.jsx]: Thanks!

**Comment by michelleweber** [comment]
Can we tighten up the first part of the copy so we can add a little more detail in the second about what exactly gets displayed where? Am thinking along the lines of:

This September, millions of us will take to the streets to demand an end to the age of fossil fuels. Show your solidarity on your site by displaying a climate strike banner for the month. On September 20th, the day of the Global Climate Strike, the banner will expand to a full-screen overlay that site visitors can dismiss.

**Pull request merged**